### PR TITLE
few refactor and added comments

### PR DESCRIPTION
**Pull request by belwase**

### CODE REVIEW

File: upwork-devs/ashish-belwase/virusshare_poc/src/file_service.py [hunk 1]
```diff
@@ -1,64 +1,100 @@
...
 from src.constants import download_path
 
+logger = logging.getLogger("GW:file_service")
+
 
 class FileService:
 
     UPLOAD_DIR = os.environ.get("STATIC_PATH", "src/static")
 
+    @staticmethod
```
Comment (susilupwork) [inline upwork-devs/ashish-belwase/virusshare_poc/src/file_service.py:15]: The method is just getting extension and name of file, does not sound correct. Please get clarification on meta information expectation from @manish2kcoder  or @dinis-cruz-gw .

**Review by susilupwork — APPROVED** [review]
Please collaborate with @samarthdd  to get proper Minio service  and integrate.

**Comment by susilupwork** [comment]
Test File distribution API should be independent of other part of the code and should run in separate K8 pod. It will call Minio service to get the file and delivery to the requesting user. The code to be independent and isolated from Scrapping part.

**Review by susilupwork — CHANGES_REQUESTED** [review]
Distribution API code be separated completely, Now it is intermingled with POC scrapper code. Please take care of this.

**Review by DinisCruz — CHANGES_REQUESTED** [review]
108 files doesn’t look right, should all these files be here?

**Pull request merged**